pages/overview/what-is-jsonschema.md

```diff
@@ -6,36 +6,108 @@ title: What is JSON Schema?
+    * You can use `type` to specify what types of data are valid for a given key. You can specify either a single type, like `string`, or an array of possible types, like `string`, `number`, or `integer`.
+    * You can also enforce formats using regular expressions with the `pattern` keyword.
+    * There are countless ways to describe specific data types and formats that your data must follow.
 
-After you define your schema, you validate your JSON documents against the schema. To do this, you can use any supported [validator](https://json-schema.org/implementations.html#validators) in whatever language or context fits your project. There are JSON Schema validation tools for nearly every modern programming language, as well as language-agnostic command-line and browser tools.
+2. **Create the schema**: Use the defined rules to create a JSON Schema that represents the expected structure and format of your JSON data.
 
-When you have a defined schema and valid JSON data, the possibilities are endless. You can do additional data processing, set up error handling, and integrate your validated data with APIs, other applications, and business logic.
+    For example, let the following file be `schema.json`:
+
+    ```
+    schema = {
+        "type": "object",
+        "properties": {
+            "name": {
+            "type": ["string", "null"],
+            "pattern": "^[A-Za-z]+$"
+            },
+            "age": {
+            "type": "integer"
+            },
+            "email": {
+            "type": "string"
+            }
+        },
+        "required": ["name", "age", "email"]
+    }
+    ```
+
+    * In the above schema, `name` can either be a `string` or `null`, `age` can only be an `integer`, `email` can only be a `string`, and the entire schema is an `object`.
+    * Here, `name` can only contain alphabetical characters. 
+    * You can make some properties mandatory to enter by using the `required` keyword. Here, `name`, `age`, and `email`, all are required to create an instance of the schema.
+
+3. **Choose a validator**: Select a JSON Schema [validator](https://json-schema.org/implementations.html#validators) that is supported in your chosen programming language or context. There are JSON Schema validation tools for nearly every modern programming language, as well as language-agnostic command-line and browser tools.
+
+4. **Validate the documents**: Use the selected validator to validate your JSON documents against the created JSON Schema.
```

Comment:
I think we should keep consistency with the glossary of terms and for this section I suggest we focus in 3 elements:  schema, keywords and instance.
* Keywords and Schema are used in step 1.
* Here I suggest to use instance instead of document.